crates/precompile-evm-balances-erc20/ERC20.sol

```diff
@@ -0,0 +1,117 @@
+// SPDX-License-Identifier: UNLICENSED
+
+pragma solidity >=0.7.0 <0.9.0;
+
+/**
+ * @title ERC20 Interface
+ *
+ * An interface exposing native eHMND tokens as ERC20 tokens.
+ *
+ * Address: 0x0000000000000000000000000000000000000802
```

Comment:
Does this value match the one here https://github.com/humanode-network/humanode/pull/746/files#r1306311603?